lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -239,9 +240,11 @@ def message_from(success, response)
         return response['message'] if response['message']
 
         message = nil
-
         if error = response.dig('status_details', 'error')
           message = error.dig('reason', 'description')
+          if detail = error.dig('type')
```

Comment:
Nitpick, but since it's only one layer into the hash, I think this could just be `if detail = error['type']` if you wanted.